release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1301,6 +1474,55 @@ This update corrects the accepted label specification so that the plug-in import
 
+[discrete]
+[id="ocp-4-10-monitoring-bug-fixes"]
+==== Monitoring
+
+* Previously, during {product-title} upgrades, the Prometheus service could become unavailable because either two Prometheus pods were located on the same node or the two nodes running the pods rebooted during the same interval. 
+This situation was possible because the Prometheus pods had soft anti-affinity rules regarding node placement and no `poddisruptionbudgets` resources provisioned. 
+Consequently, metrics were not collected and rules were not evaluated over a period of time.
++
+To fix this issue, the Cluster Monitoring Operator (CMO) now configures hard anti-affinity rules to ensure that the two Prometheus pods are scheduled on different nodes.
+The CMO also provisions `poddisruptionbudgets` resources to ensure that at least one Prometheus pod is always running.
+As a result, during upgrades, the nodes now reboot in sequence to ensure that at least one Prometheus pod is always running. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1933847[*BZ#1933847*])
+
+* Previously, the Thanos Ruler service would become unavailable when the node that contains the two Thanos Ruler pods experienced an outage. 
+This situation occurred because the Thanos Ruler pods had only soft anti-affinity rules regarding node placement. 
+Consequently, user-defined rules would not be evaluated until the node came back online.
++
+With this release, the Cluster Monitoring Operator (CMO) now configures hard anti-affinity rules to ensure that the two Thanos Ruler pods are scheduled on different nodes. 
+As a result, a single-node outage no longer creates a gap in user-defined rule evaluation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955490[*BZ#1955490*])
+
+* Previously, the Prometheus service would become unavailable when the two Prometheus pods were located on the same node and that node experienced an outage.
+This situation occurred because the Prometheus pods had only soft anti-affinity rules regarding node placement.
+Consequently, metrics would not be collected, and rules would not be evaluated until the node came back online.
++
+With this release, the Cluster Monitoring Operator configures hard anti-affinity rules to ensure that the two Prometheus pods are scheduled on different nodes.
+As a result, Prometheus pods are now scheduled on different nodes, and a single node outage no longer creates a gap in monitoring.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1949262[*BZ#1949262*])
+
+* Previously, during {product-title} patch upgrades, the Alertmanager service might become unavailable because either the three Alertmanager pods were located on the same node or the nodes running the Alertmanager pods happened to reboot at the same time. 
+This situation was possible because the Alertmanager pods had soft anti-affinity rules regarding node placement and no `poddisruptionbudgets` provisioned. 
```

Comment:
```suggestion
This situation was possible because the Alertmanager pods had soft anti-affinity rules regarding node placement and no `PodDisruptionBudget` provisioned. 
```

Reply:
fixed